src/containers/pages/IRSLite/MapLite/tests/index.test.tsx

```diff
@@ -494,6 +494,12 @@ describe('components/IRS Reports/IRSLiteReportingMap', () => {
         </Router>
       </Provider>
     );
+
+    //  map or error components should load when all the promises are resolved
+    wrapper.update();
+    expect(wrapper.find('MemoizedGisidaLiteMock').length).toEqual(0);
+    expect(wrapper.find('ErrorPage').length).toEqual(0);
```

Comment:
How come these are both not present?  Trying to understand how the test works with respect to the comment here.

Reply:
My expectation is, the map and error components should only be loaded after the requests from the server have been finalized. If they show after the requests are through then the error is genuine.